### ISSUE TITLE
learning curve of training loss

### ISSUE THREAD

**Issue by LeeKeyu** [hehao13/CameraCtrl#29]
Hi authors, thanks for your impressive work.
I tried to train the I2V camera control model using 180*288 resolution on 8 gpus, using batch size=1 and did not change other parts of your config. But the training loss seems to remain at about 0.25 throughout the 50000 iters. The final results look noisy. Could you please provide the training curve of your model for a reference?
Thanks a lot for your help!

**Comment by Hippogriff** [comment]
+1
